arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -254,20 +255,23 @@ public virtual async Task<GetEdgeCollectionsResponse> GetEdgeCollectionsAsync(st
         /// <param name="collectionName"></param>
         /// <param name="vertex"></param>
         /// <param name="query"></param>
+        /// <param name="serializationOptions">The serialization options. When the value is null the
+        /// the serialization options should be provided by the serializer, otherwise the given options should be used.</param>
         /// <returns></returns>
         public virtual async Task<PostVertexResponse<T>> PostVertexAsync<T>(
             string graphName,
             string collectionName,
             T vertex,
-            PostVertexQuery query = null)
+            PostVertexQuery query = null,
+            ApiClientSerializationOptions serializationOptions = null)
         {
             string uri = _graphApiPath + '/' + WebUtility.UrlEncode(graphName) +
                 "/vertex/" + WebUtility.UrlEncode(collectionName);
             if (query != null)
             {
                 uri += "?" + query.ToQueryString();
             }
-            var content = GetContent(vertex, false, false);
+            var content = GetContent(vertex, serializationOptions);
```

Comment:
Can you verify that this needs as input serializationOptions  or we should use the default GetContent(vertex, false, false); ?

Reply:
It is needed yes 👍  `T vertex` is a document for a vertex collection.